chef/cookbooks/provisioner/recipes/dhcp_update.rb

```diff
@@ -21,9 +21,9 @@
              'if option arch = 00:06 {
        filename = "discovery/ia32/efi/bootia32.efi";
      } else if option arch = 00:07 {
-       filename = "discovery/x86_64/efi/bootx64.efi";
+       filename = "discovery/x86_64/efi/boot/grub2/x86_64-efi/core.efi";
      } else if option arch = 00:09 {
-       filename = "discovery/x86_64/efi/bootx64.efi";
+       filename = "discovery/x86_64/efi/boot/grub2/x86_64-efi/core.efi";
      } else if option arch = 00:0b {
        filename = "discovery/aarch64/efi/bootaa64.efi";
```

Comment:
I had some issues with building image using `grub2-mknetdir`, I looked once again for some more information about this. We can check and building setup with information, which I have found (putted in Trello card).
I will open bugreport against `grub2-mknetdir` and ARM support.